include/language.php

```diff
@@ -331,6 +331,21 @@ public function __get( $property ) {
 			return $this->{$url_getter}();
 		}
 
+		if ( 'mo_id' === $property ) {
+			trigger_error( // phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_trigger_error
```

Comment:
There's a dedicated method to trigger errors for deprecated props, see `$this->deprecated_property()`.

Reply:
I see, don't know if it worth making `deprecated_property()` less specific.

Reply:
Is it worth? Why someone would like to access to this property? To access strings translations, we pass the complete language to `PLL_MO` methods..